### ISSUE TITLE
Gas clouds in Nagrand - unable to suck some of them

### ISSUE THREAD

**Issue by Annamaria-CC** [azerothcore/azerothcore-wotlk#17148]
https://github.com/chromiecraft/chromiecraft/issues/5953
### What client do you play on?

enGB

### Faction

Both

### Content Phase:

70

### Current Behaviour

Gas clouds in Nagrand next to floating islands in the air are often to far away to suck.
Mostly all of the ground ones are fine.

### Expected Blizzlike Behaviour

Should be able to suck em

### Source

![air clouds nagrand](https://github.com/chromiecraft/chromiecraft/assets/98283901/4cfb7c65-526d-4211-aa9c-ff8f6624aaa6)
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/20357406/88972b29-cad4-4e37-9333-743a8437e86a)

### Steps to reproduce the problem

Just fly to Nagrand - Telaar usually have 1 or 2 just outside thats often unsuckable

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/30197f40d687eae97a07d6730b6933c63f1a3983

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by elthehablo** [comment]
Yup these are really annoying. Would be nice if someone were to sniff them. Also curious to what movement they should have. Some have wander, some have wps in our db

**Comment by Annamaria-CC** [comment]
didnt notice them moving on wotlk ptr but didnt check for that specifically.

**Comment by elthehablo** [comment]
> didnt notice them moving on wotlk ptr but didnt check for that specifically.

That would make things a lot easier. But obviously gonna need spawn points sniffed

**Comment by chauusie** [comment]
There's a lot of historic comments on wowhead about people unable to reach them https://www.wowhead.com/npc=24222/windy-cloud so maybe it's not a bug? Unless it is worse here than it was in OG TBC?
